src/frontend/js/components/SaleTunnel/WithdrawRightCheckbox/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { Alert, Checkbox, VariantType } from '@openfun/cunningham-react';
+import { useCallback, useEffect, useState } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl/lib';
+import { useSaleTunnelContext } from 'components/SaleTunnel/GenericSaleTunnel';
+
+const messages = defineMessages({
+  waiveCheckboxExplanation: {
+    defaultMessage:
+      'This training will start before the end of your withdrawal period. You must waive your right of withdrawal to subscribe.',
```

Comment:
Maybe we could simplify a bit this explanation:
```suggestion
      'This training will start before the end of your withdrawal period. You must waive it to subscribe.',
```